### PR TITLE
pass --cacert flags to `helm repo` commands too

### PR DESCRIPTION
**Pull request by orymate**
* https://github.com/k3s-io/helm-controller/issues/89

The job fails otherwise with the following:

```
+ helm_v3 repo add x 'https://x'
Error: looks like "https://x" is not a valid chart repository or cannot be reached: Get "https://x/index.yaml": x509: certificate signed by unknown authority
```

### CODE REVIEW

File: entry [hunk 1]
```diff
@@ -109,15 +109,15 @@ helm_repo_init() {
 
 	if [[ "${HELM}" == "helm_v3" ]]; then
 		if [[ ${CHART} == stable/* ]]; then
-			${HELM} repo add stable ${STABLE_REPO_URL}
+			${HELM} repo ${CA_FILE_ARG} add stable ${STABLE_REPO_URL}
```
Comment (brandond) [inline entry:112]:
```suggestion
			${HELM} repo add stable ${CA_FILE_ARG} ${STABLE_REPO_URL}
```

File: entry [hunk 2]
```diff
@@ -109,15 +109,15 @@ helm_repo_init() {
...
 			${HELM} repo update
 		fi
 	else
 		${HELM} repo update --strict || ${HELM} repo remove stable
 	fi
 
 	if [[ -n "${REPO}" ]]; then
-		${HELM} repo add ${NAME%%/*} ${REPO}
+		${HELM} repo ${CA_FILE_ARG} add ${NAME%%/*} ${REPO}
```
Comment (brandond) [inline entry:120]:
```suggestion
		${HELM} repo add ${CA_FILE_ARG} ${NAME%%/*} ${REPO}
```

**Review by brandond — CHANGES_REQUESTED** [review]
Thanks for this! I absolutely had done it this way on my local branch, but I forgot to push it before merging.

Would you mind also removing the `${CA_FILE_ARG}` from the other commands? It is only actually necessary on the `repo add` commands.

**Review by brandond — APPROVED** [review]

**Comment by orymate** [comment]
Updated as requested. It seems to work based on a quick test.

**Review by briandowns — APPROVED** [review]

**Pull request merged**